### DISCUSSION TITLE
What is "Request with Backend?"

### DISCUSSION THREAD

**Discussion by joelouthan** [anse-app/anse#106] [Q&A] [answered]
I am curious what this switch does.

**Comment by ddiu8081** [comment, marked as answer]
By default, all requests connect directly from the browser to the backend service (e.g. api.openai.com).

However, there are some countries with network blocking (e.g. China) where direct connection to OpenAI's service is not possible.

Turning this option on will let the request go through Anse proxy and then forward it to the backend, to solve the possible direct connection problem.

**Comment by joelouthan** [comment]
That makes sense. I don't need it. Thank you.
